pkg/cmd/server/api/helpers.go

```diff
@@ -26,6 +27,18 @@ func init() {
 	for _, feature := range KnownOpenShiftFeatures {
 		knownOpenShiftFeatureSet[strings.ToLower(feature)] = feature
 	}
+	for alias, feature := range FeatureAliases {
+		knownOpenShiftFeatureSet[strings.ToLower(alias)] = feature
+	}
+}
+
+// NormalizeOpenShiftFeature returns canonical name for given OpenShift feature
+// flag or an alias. Returns empty string if nothing matches.
```

Comment:
all the places this is used, the returned bool is now checked as the signal for an unknown item, so returning the lowercased name should be fine